src/templates/index.js

```diff
@@ -1,5 +1,4 @@
 module.exports = {
-  home: require("./home"),
```

Comment:
The home page now uses the default template, since it no longer needs to list every page with search being completed.